web-platform-when-why/index.html

```diff
@@ -0,0 +1,423 @@
+              of 2015, mobile Web usage is increasing without affecting desktop usage. The opportunity is
+              not shifting the focus but going multiplatform. HTML5 is already universal. It is
+              multiplatform for free.
+            </aside>
+          </section>
+
+          <section>
+            <blockquote>
+              <p>Confluence of factors is driving the huge uptick in app usage time, and apps are overtaking other media in importance. However, it’s also important to recognize the others haven’t gone away and the opportunity lies in multi-platform engagement.</p>
+              <p style="text-align: right;">&mdash;2015 U.S. Mobile Report Opportunities Key Points</p>
+            </blockquote>
+          </section>
+
+          <section>
+            <h3>Lack of friction</h3>
+            <img alt="Every step costs you 20% of users" src="./imgs/step-cost-20.png" style="width: 70%;"/>
+            <aside class="notes">
+              The other benefit is actually the great differentiator. The Web spreads through the Internet
+              which is not a propietary marketplace but a completely free information network. The URL is
+              viral, anyplace in the Internet can point you to any Web application without never facing an
+              installation. You click the link, you get working content.
+            </aside>
+          </section>
+        </section>
+
+        <section>
+          <section>
+            <h2>So, what's stopping the Web?</h2>
+            <aside class="notes">
+              Well, there are obvious reasons to not choose the web. For instance, if the platform does
+              not cover your API needs but, event when this is not a problem and having these both
+              interoperability and lack of friction features, so exclussive and so fetching the Web does
+              not seem to fit our needs. Why?
+            </aside>
+          </section>
+
+          <section>
+            <p><strong>User experience</strong></p>
+            <aside class="notes">
+              The answer is user experience. In the beginning, the Web was never designed for mobile usage.
+              We surfed the web by paying special attention to the browser, in desktop, powerful multicore
+              setups with big screens an a reliable network wired connection.
+            </aside>
+          </section>
+
+          <section>
+            <h3>Dependence on connectivity</h3>
+            <img alt="Dinosaur game in Chrome" src="./imgs/dinosaur-game-chrome.jpg" style="width: 70%;"/>
+            <aside class="notes">
+              In the mobile realm you can not rely on be always connected. If the connectivity is lost,
+              the experience is broken. The complete application vanishes. Of course a user can understand
+              an online application to not show content if offline but breaking with no information? I
```

Comment:
a user can understand **for** an online application **not to** show content if it **goes** offline, but breaking **down** with